_source/user-guide/alerts/configure-alerts.md

```diff
@@ -10,101 +10,166 @@ contributors:
+Your query is an important source of information
+when you're investigating a triggered alert.
+Shorten your query by using filters whenever you can.
+
+If you use an invalid query,
+the alert will be automatically disabled.
+To make sure it's valid,
+test your query in Kibana.
 {:.info-box.important}
 
-##### _(Optional)_ Edit group by settings
+###### Group by
 
-![Group alert fields]({{site.baseurl}}/images/alerts/alerts--group-by.png)
+![Alert group by settings]({{site.baseurl}}/images/alerts/alerts--group-by.png)
 
-Click **Add group by** to add up to 3 groups.
+Moving down, you'll come to the **Group by** setting next.
 
-In the **Select Field** list,
-choose a field to group by.
+With _Group by_, you can choose up to 3 fields to group.
+The alert will return the aggregated results for each group.
 
-To limit the available fields,
-choose a log type from the **Filter by type** list.
-To show fields for all log types, choose **Clear filter**.
+Careful, though: The order of these fields matters.
+Results are grouped by
+the leftmost field, then the center field, then the right field.
+In other words, the image above groups results by
+continent, then country, then city.
+If we had reversed the order (city, then country, then continent),
+it would likely generate unintended results.
 
-##### Set threshold and severity levels
+###### Accounts to search
+
+![Accounts to search setting]({{site.baseurl}}/images/alerts/accounts-to-search.png)
+
+Next you'll choose the _Accounts to search_.
+
+If you choose **All accounts**,
+this alert will query the logs in all the accounts it has access to.
+This will include future accounts.
 
-Set your threshold and severity levels in the **Trigger** section.
-Click **Add multiple conditions** to add up to 3 threshold levels.
+To limit the alert to a set list of accounts,
+click **Just these accounts**,
+and add the accounts you want.
+This list is always limited to your selection,
+so you'll need to manually update the list if you need it to change.
+
+##### Set threshold and severity levels
 
 ![Alert trigger thresholds]({{site.baseurl}}/images/alerts/alerts--trigger-settings.png)
 
-##### _(Optional)_ Set recipients
+Set your threshold and severity levels in the _Trigger if..._ section.
+
+Click **Add a threshold** to set up to 5 threshold levels,
+each with its own severity tag.
+
+##### _(Optional)_ Set notification details
+
+###### Description and tags
+
+![Alert description and tags]({{site.baseurl}}/images/alerts/description-and-tags.png)
 
-If you want to receive notifications or emails when the alert is triggered,
-choose an alert endpoint.
-If you don't choose an endpoint,
-triggered alerts will still be logged:
+The **Description** is visible on the _Alert definitions_ page.
+It's also included with emails and Slack messages when the alert is triggered.
+We recommend making your description helpful to recipients,
+like telling them how to fix the issues that led to the alert.
```

Comment:
We have a char limit for the description box

Reply:
Even less likely users will hit it. Leaving this as is.